### ISSUE TITLE
MPRIS issue

### ISSUE THREAD

**Issue by mtlhdnrd** [lawfx/YoutubeNonStop#42]
I am on arch linux with i3wm, and I use https://github.com/altdesktop/playerctl for media control. however, when listening to music, after ~10 seconds of leaving the yt music tab, play/pause and next/prev stop working. when I try to pause the playback, the tab's name changes back to "Youtube Music" from the song's title, but it keeps playing. then when I go back to yt music, and click pause, it pauses then immediately starts playing again. when I disable this add-on, this issue disappears.
firefox version: 93.0
kernel version: 5.14.9-arch2-1
youtube non-stop version: 0.9.0
playerctl version: 2.4.1

**Comment by JohnyP36** [comment]
I guess https://github.com/lawfx/YoutubeNonStop/releases/tag/v0.9.1 solves your problem.

**Comment by mtlhdnrd** [comment]
Yes, it fixed it. When I opened this issue, the latest version was 0.9.0.
